src/operator/bufferToggle.ts

```diff
@@ -35,8 +37,8 @@ import {errorObject} from '../util/errorObject';
  *
  * @param {Observable<O>} openings An observable of notifications to start new
  * buffers.
- * @param {function(value: O): Observable} closingSelector A function that takes
- * the value emitted by the `openings` observable and returns an Observable,
+ * @param {function(value: O): SubscribableOrPromise} closingSelector A function that takes
+ * the value emitted by the `openings` observable and returns an Subscribable or Promise,
```

Comment:
Can you keep the column limited under 80 characters? Also `an Subscribable` => `a Subscribable`.